packages/wdio-local-runner/src/worker.ts

```diff
@@ -108,12 +134,12 @@ export default class WorkerInstance extends EventEmitter {
         /**
          * handle debug command called within worker process
          */
-        if (payload.origin === 'debugger' && payload.name === 'start') {
+        if (childProcess && payload.origin === 'debugger' && payload.name === 'start') {
```

Comment:
```suggestion
        if (payload?.origin === 'debugger' && payload?.name === 'start') {
```

Reply:
it's

`childProcess && payload.origin === 'debugger'`
not
`payload && payload.origin === 'debugger'`